pages/installation_quick_start.md

```diff
@@ -4,65 +4,52 @@ title: Installation Quick Start
 order: 2
 ---
 
-MDAnalysis offers two methods to install the released version. For most
-users we recommend the [*Conda*](#conda) installation method.
+MDAnalysis can be installed using [mamba][] (recommended) or [pip][].
 
-## Conda ##
+## Mamba ##
 
-If you don't have [conda][] installed yet, follow the [installation
-instructions for conda][].
-
```

Comment:
I think we should still have links to installing mamba/miniforge (see https://github.com/MDAnalysis/UserGuide/issues/361#issuecomment-2176431603 ) and conda. There are people who don't use it every day. It's important to give some minimal guidance. It's also a bit odd to recommend mamba but requiring people to somehow have conda already installed.